### PR TITLE
use stable httplug

### PR DESCRIPTION
**Pull request by krichprollsch**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,9 +12,9 @@
     ],
     "require": {
         "php": ">=5.4",
-        "php-http/httplug": "^1.0@beta",
-        "php-http/client-common": "~0.1",
-        "php-http/discovery": "~0.5"
+        "php-http/httplug": "^1.0",
+        "php-http/client-common": "^1.0",
+        "php-http/discovery": "^0.5"
```
Comment (sagikazarmark) [inline composer.json]: This should be updated to 0.7

**Comment by sagikazarmark** [comment]
Thanks @krichprollsch . Can you pleas update discovery version as well?

File: composer.json (hunk 1 above)
Comment (krichprollsch) [inline composer.json]: updated now

**Comment by krichprollsch** [comment]
Hum discovery 0.7 is depend of a beta version of puli/composer-plugin :/
WDYT @sagikazarmark ?

**Comment by sagikazarmark** [comment]
Yeah, these lines are missing from composer.json to allow that dependency:

https://github.com/php-http/guzzle6-adapter/blob/master/composer.json#L49-L50

**Comment by krichprollsch** [comment]
yeah, tests are green now, ty!

**Comment by sagikazarmark** [comment]
Thanks

**Comment by krichprollsch** [comment]
@sagikazarmark have you planned to create a tag ? I would like to update my deps...
Or you wait the stable release ?

**Comment by sagikazarmark** [comment]
I can tag an MJZ version if you would like. I would wait the stable release, until php-http/discovery#49 is resolved.

**Comment by krichprollsch** [comment]
I will be very happy to have a tag yes :) ty!

**Comment by sagikazarmark** [comment]
Tagged v0.2.0

**Pull request merged**